packages/react/package.json

```diff
@@ -65,7 +65,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@slashid/slashid": "3.25.0",
+    "@slashid/slashid": "3.26.1",
```

Comment:
Please remember to update the peer dependency too, as this won't work unless the core SDK is installed as well

Reply:
Good point! Done https://github.com/slashid/javascript/pull/264/commits/20f11d3154f0fffc7ebe42ad4611e6f22bc9c763